config/initializers/devise.rb

```diff
@@ -249,7 +249,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
The current logout button doesn't work as the link is making a get request. This should fix it.